cmd/crank/beta/xpkg/xpkg.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2023 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package xpkg contains beta Crossplane packaging commands.
+package xpkg
+
+// Cmd contains commands for interacting with xpkgs.
+// TODO(lsviben) add the rest of the commands from up (batch, xpextract).
+type Cmd struct {
+	Init    initCmd    `cmd:"" help:"Initialize a package from a template."`
+	Install installCmd `cmd:"" help:"Install a package."`
+	Update  updateCmd  `cmd:"" help:"Update an installed package."`
+}
+
+// Help prints out the help for the xpkg command.
+func (c *Cmd) Help() string {
+	return `
+A Crossplane package is an opinionated OCI image that contains an additional layer
+holding meta information to drive the Crossplane package manager. The package manager
+uses this information to install packages into a Crossplane instance.
+
+Furthermore, a Crossplane package may contain meta information that describes
+how to represent the package in a user interface. This information is used by
+the Upbound marketplace to display packages and their contents. See the xpkg
+reference document for more information.
+
+There are different kinds of Crossplane packages, each with a different set of
+meta information and files in the additional layer. The following kinds are 
+currently supported:
+
+- **Provider**: A Crossplane package that contains a Crossplane provider. The layer
+  contains a crossplane.yaml file with a "meta.pkg.crossplane.io/v1alpha1"
```

Comment:
update this version to v1? and make sure the same updates are made to `cmd/crank/xpkg/xpkg.go` too?